apps/charterafrica/src/payload/fields/toolSourceField.js

```diff
@@ -0,0 +1,21 @@
+const source = () => ({
```

Comment:
Need to pick one naming scheme and stick to it; with or without Field at the end e.g. `toolSource` or `toolSourceField`.